test/configCases/issues/issue-9206-app/webpack.config.js

```diff
@@ -0,0 +1,14 @@
+var webpack = require("../../../../");
+
+module.exports = {
+	mode: "development",
+	entry: ["./index.js"],
+	plugins: [
+		new webpack.DllReferencePlugin({
+			name: "../issue-9206-vendor/bundle0.js",
+			scope: "issue-9206-vendor",
+			sourceType: "commonjs2",
+			manifest: require("../../../js/config/issues/issue-9206-vendor/manifest0.json")
```

Comment:
Test cases are executed in alphabetical order, so to make vendor to execute before app you may name it `issue-9206-0-vendor` and `issue-9206-1-app`